packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -71,7 +85,7 @@ abstract class SemanticsEvent {
 class AnnounceSemanticsEvent extends SemanticsEvent {
 
   /// Constructs an event that triggers an announcement by the platform.
-  const AnnounceSemanticsEvent(this.message, this.textDirection)
+  const AnnounceSemanticsEvent(this.message, this.textDirection, this.assertivenessSetting)
```

Comment:
Should we make this an optional named parameter that defaults to polite? My thinking is that most users should use polite and only if they have an actual use case should consider (and look into) the assertive setting.